app/javascript/components/App.js

```diff
@@ -17,6 +18,7 @@ class App extends Component {
     super(props);
     this.state = {
       apartments: mockApartments,
+      users: mockUsers,
```

Comment:
We actually do not need to mock users at all because we have devise handling all of our user processes.